src/__data__/inAppMessages.ts

```diff
@@ -1,165 +1,172 @@
-        },
-        bottom: {
-          displayOption: 'AutoExpand'
-        },
-        left: {
-          percentage: 0
-        }
+      bottom: {
+        displayOption: 'AutoExpand'
       },
-      webInAppDisplaySettings: {
-        position: 'Center'
+      left: {
+        percentage: 0
       }
     },
-    customPayload: {
+    webInAppDisplaySettings: {
+      position: 'Center'
+    }
+  },
+  customPayload: {},
+  trigger: {
+    type: 'immediate'
+  },
+  saveToInbox: true,
+  inboxMetadata: {
+    title: 'hey there',
+    subtitle: 'mensaje',
+    icon: ''
+  },
+  priorityLevel: 300.5,
+  read: false
+};
+
+const expiredMessage: InAppMessage = {
+  messageId: 'expiredMessage!',
+  campaignId: 2641072,
+  createdAt: 1627320168381,
+  expiresAt: 1635096168381,
+  content: {
+    html: `<html>\n<head>\n\t<title></title>\n\t<meta name="viewport" content="width=device-width, 
+    initial-scale=1, viewport-fit=cover"/>\n\t
+    <link href="https://storage.googleapis.com/publicstatic.hellobrigit.com/fonts.css" 
+    rel="stylesheet" />\n\t
+    <link href="https://storage.googleapis.com/publicstatic.hellobrigit.com/iterable-templates.css" 
+    rel="stylesheet" />\n</head>\n<body>\n<div class="overlay"></div>\n\n
+    <div class="modal-container" style="margin: auto; width: 70%; 
+    border: 1px dashed black; padding: 10px;background:#ffffff">\n<h2>Hey ,</h2>\n\n
+    <p>This is a sample In App message, how are you?</p>\n
+    <a href="iterable://dismiss">close</a> 
+    <a href="action://close-second-iframe">profile</a>
+    <input type="text" /></div>\n</body>\n</html>\n`,
+    payload: {
       eventName: 'DailyRewardScreen',
       event: 'screen'
     },
-    trigger: {
-      type: 'immediate'
-    },
-    inboxMetadata: {
-      title: 'hey there',
-      subtitle: 'mensaje',
-      icon: ''
+    inAppDisplaySettings: {
+      top: {
+        displayOption: 'AutoExpand'
+      },
+      right: {
+        percentage: 0
+      },
+      bottom: {
+        displayOption: 'AutoExpand'
+      },
+      left: {
+        percentage: 0
+      }
     },
-    saveToInbox: false,
-    priorityLevel: 300.5,
-    read: false
+    webInAppDisplaySettings: {
+      position: 'Center'
+    }
+  },
+  customPayload: {
+    eventName: 'DailyRewardScreen',
+    event: 'screen'
   },
-  {
-    messageId: 'xEA5MZmz8T2pAoi1VJ5hRuDuazdZzXS9BD85t00w0r9ax',
-    campaignId: 2632531,
-    createdAt: 1627332441416,
-    expiresAt: 1634836811283,
-    content: {
-      html: `<html>\n<head>\n\t<title></title>\n\t<meta name="viewport" content="width=device-width, 
+  trigger: {
+    type: 'immediate'
+  },
+  inboxMetadata: {
+    title: 'hey there',
+    subtitle: 'mensaje',
+    icon: ''
+  },
+  saveToInbox: false,
+  priorityLevel: 300.5,
+  read: false
+};
+
+const previouslyCachedMessage: InAppMessage = {
+  messageId: 'previouslyCachedMessage!',
+  campaignId: 2632531,
+  createdAt: 1627332441416,
+  expiresAt: 1670355237175,
```

Comment:
updated value (expired messages get immediately removed from the cache so should test with unexpired message)